src/modbus-tcp-client.js

```diff
@@ -146,14 +154,14 @@ module.exports = stampit()
 
             reqId += 1;
 
-            var head = Buffer.allocUnsafe(7)
+            var head = Buffer.allocUnsafe(7);
 
-            head.writeUInt16BE(reqId, 0)
-            head.writeUInt16BE(this.protocolVersion, 2)
-            head.writeUInt16BE(pdu.length + 1, 4)
-            head.writeUInt8(this.unitId, 6)
+            head.writeUInt16BE(reqId, 0);
```

Comment:
ahhh @BauchBeinePoe you prefer semicolon? Sorry I'll try to remember it. we are working with the `standard` linter recommendations (https://www.npmjs.com/package/standard) and dropped semicolons internally. just not used to it.